kafka/conn.py

```diff
@@ -54,6 +54,21 @@ class SSLWantReadError(Exception):
     class SSLWantWriteError(Exception):
         pass
 
+# needed for SASL_GSSAPI authentication:
+disable_gssapi = False
```

Comment:
mild preference for just setting `gssapi = None` on ImportError. See kafka/codec.py